hq/test/aws/support/TrustedAdvisorSGOpenPortsTest.scala

```diff
@@ -65,4 +68,57 @@ class TrustedAdvisorSGOpenPortsTest extends FreeSpec with Matchers with AttemptV
       }
     }
   }
+
+  "sorting" - {
```

Comment:
It looks like this tests that the three separate sorts all work individually, but not that they are themselves correctly sorted. I realise it's a tricky test to write but it'd be good to know that e.g. unused results always go below used ones.

Reply:
ok, i'll try to add one more test case with used and unused security groups.